provider/cluster/kube/k8s_integration_test.go

```diff
@@ -5,22 +5,21 @@ package kube
 import (
 	"bufio"
 	"context"
-	"fmt"
-	"math/rand"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/ovrclk/akash/testutil"
 )
 
 func TestNewClient(t *testing.T) {
 	ctx := context.Background()
 
-	ns := fmt.Sprintf("test-provider-cluster-kube-client-%v", rand.Uint32())
+	ns := "effectively unused NS variable"
```

Comment:
@Ropes the tests are failing because this is not a valid namespace.

```
Namespace "effectively unused NS variable" is invalid: metadata.name: Invalid value: "effectively unused NS variable": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```

Reply:
This branch needs to be rebased on to the network policies, I've fixed it. That variable was effectively creating  a k8s Namespace, but then it wasn't used for anything, and wasn't getting cleaned up. The current `master` behavior actually creates two namespaces, but only uses one.

Reply:
There's one to store manifests and others to run workloads in.

Reply:
The `leases` namespace stores Manifests, the `test-provider...` namespace was created for just the integration test, but wasn't cleaned up because it was out of the path of normal cleanup.